x-pack/plugins/security_solution/server/lib/timeline/saved_object.ts

```diff
@@ -95,9 +99,27 @@ export interface Timeline {
 
 export const getTimeline = async (
```

Comment:
Seems that we have `getTimeline` and `getTimelineByTemplateTimelineId` doing the same thing, I am thinking of having a follow up PR to do a clean up for that. So we can just keep `getTimeline` in our code, what do you think?